submissions/andrewklmn/a-tiny-js-world/index.js

```diff
@@ -65,12 +126,11 @@ const inhabitants = [
   man,
 ];
 
-const getFormatedOutput = ({species, name, gender, legs, hands, saying, friends}) =>  {
-  if(typeof(friends) === 'object'){
-      friends = friends.map(friend => friend.name).join(', ');
-  }
-  return `${species}; ${name}; ${gender}; ${legs}; ${hands}; ${saying}; ${friends}`;
-}
+// define friends for everyone except the cat...
+dog.friends = [man, woman, cat];
```

Comment:
Can be good idea use setters from ES6 or make your own setter `setFriends`.

If you have used classes - use their contract.